CHANGELOG.md

```diff
@@ -25,6 +24,7 @@
 - Timestamp has millis precision.
 - For better groupping, add your own package to the `addInAppInclude` list, e.g.  `options.addInAppInclude('sentry_flutter_example');`
 - A few classes of the `Protocol` were renamed.
+- Sentry.init() now needs a callback argument which must run the host app after Sentry initialization. 
```

Comment:
I'd suggest adding a breaking change section on the version above (the one that is gonna be published)

Reply:
✅